src/Hubbup.Web/Client/Standup/components.tsx

```diff
@@ -0,0 +1,370 @@
+import * as React from 'react';
+
+import * as Data from '../data';
+
+const fetchSettings: RequestInit = {
+    credentials: "same-origin",
+    headers: {
+        ["X-Requested-With"]: "XMLHttpRequest"
```

Comment:
Also you don't need the square brackets around `"X-Requested-With"` since they are only needed when evaluating dynamic expressions.